.changeset/breezy-chairs-relax.md

```diff
@@ -0,0 +1,9 @@
+---
+"@nomiclabs/hardhat-solhint": major
+---
+
+Ensured the check task exists with exit code 1 when solhint raises any errors and updated solhint dependency to [v5.0.2](https://github.com/protofire/solhint/releases/tag/v5.0.2)
```

Comment:
Can we split this into two separate changesets.

So one for `Ensured the check task exists with exit code 1 when solhint raises any errors` and the other for update to v5.